src/app/components/Recommendations/index.tsx

```diff
@@ -32,8 +32,8 @@ const Recommendations = ({ data }: { data: Recommendation[] }) => {
     'aria-labelledby': labelId,
   };
 
-  const { hasMostRead, header: mostReadHeader } = mostRead || {};
-  const { skipLink } = recommendations || {};
+  const { hasMostRead } = mostRead || {};
+  const { skipLink, header } = recommendations || {};
```

Comment:
We're awaiting confirmation from editorial on the actual text for this, as it currently conflicts with the 'Most Read' header in the Most Read section at the bottom of the page.